eva-lib/src/main/java/uk/ac/ebi/eva/lib/metadata/eva/ArchiveEvaproDBAdaptor.java

```diff
@@ -117,6 +118,19 @@ public QueryResult getSpecies() {
         return new QueryResult(null, ((Long) (end - start)).intValue(), result.size(), result.size(), null, null, result);
     }
 
+    @Override
+    public QueryResult countVariants(List<VariantStudy> variantStudies) {
```

Comment:
This query shouldn't be done against the Oracle database, should be done against the MongoDB service. This method should be implemented at: `VariantWithSamplesAndAnnotationsService` that is on the library Variation-commons.